lib/taurus/qt/qtgui/panel/taurusmodelchooser.py

```diff
@@ -149,6 +149,8 @@ class TaurusModelChooser(TaurusWidget):
     '''
 
     updateModels = Qt.pyqtSignal('QStringList')
+    # (list<str> or QMimeData)
```

Comment:
> this signal is only used for backwards compatibility with the old AttributeChooser ...
If that is the case, then do not worry about it, since `AttributeChooser` should be removed